src/app/src/main/resources/metadata/charts/epidemic_trajectory/data.jsonata

```diff
@@ -48,8 +48,8 @@
     "hovertemplate": "%{y:,.0f}<extra></extra>"
   },
   {
-    "x": time_series.date,
-    "y": time_series.reported_deaths,
+    "x": timeSeries[$type(reportedDeaths)!='null'].date,
```

Comment:
We get null values for dates with no reported deaths, which makes the `$max` calculation in the layout break (it requires all values in the array to be numbers) - seemed easiest to filter out these values here.